src/components/augmint-ui/message/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { StyledDiv } from "./styles";
+import { StyledIcon } from "../icon/styles";
+
+export default function Message(props) {
+    const { children } = props;
+    if (props.onDismiss) {
+        return React.createElement(
+            StyledDiv,
+            props,
+            React.createElement(StyledIcon, { className: "fas fa-times close", onClick: props.onDismiss }),
+            children
```

Comment:
`React.createElement` only takes 3 parameters, the `children` will be ignored.

Reply:
no, the first and second parameters are fixed, but every other is a child.
`React.createElement(
  type,
  [props],
  [...children]
)`
https://reactjs.org/docs/react-api.html#createelement